ctest-next/tests/basic.rs

```diff
@@ -87,16 +83,32 @@ fn test_entrypoint_hierarchy() {
     let crate_path = include_path.join("hierarchy/lib.rs");
     let library_path = "hierarchy.out.a";
 
-    check_entrypoint("hierarchy.h", crate_path, library_path, include_path);
+    let (mut gen, out_dir) = default_generator(1, "hierarchy.h").unwrap();
+    check_entrypoint(&mut gen, out_dir, crate_path, library_path, include_path);
 }
 
 #[test]
-fn test_entrypoint_simple() {
+fn test_skip_simple() {
     let include_path = PathBuf::from("tests/input");
     let crate_path = include_path.join("simple.rs");
-    let library_path = "simple.out.a";
+    let library_path = "simple.out.skip.a";
 
-    check_entrypoint("simple.h", crate_path, library_path, include_path);
+    let (mut gen, out_dir) = default_generator(1, "simple.h").unwrap();
+    gen.skip_const(|c| c.ident() == "B");
+
+    check_entrypoint(&mut gen, out_dir, crate_path, library_path, include_path);
+}
+
+#[test]
+fn test_map_simple() {
+    let include_path = PathBuf::from("tests/input");
+    let crate_path = include_path.join("simple.rs");
+    let library_path = "simple.out.map.a";
+
+    let (mut gen, out_dir) = default_generator(1, "simple.h").unwrap();
+    gen.rename_constant(|c| (c.ident() == "B").then(|| "C_B".to_string()));
+
+    check_entrypoint(&mut gen, out_dir, crate_path, library_path, include_path);
 }
```

Comment:
Could you change `.skip` and `.map` to `.with-skips` and `.with-renames`? Just trying to make the outputs a bit more obvious.